R/SVG.R

```diff
@@ -9,6 +9,8 @@
 #' @param pointsize Default point size.
 #' @param standalone Produce a standalone svg file? If \code{FALSE}, omits
 #'   xml header and default namespace.
+#' @param font_aliases List of fonts, typically created with
```

Comment:
Yeah, that's a good name for the two types of font. (Of course, for system fonts you'd need the same font on the computer that generated it and the computer that viewed it, but that's up to the user to handle)